src/commons/repl/Repl.tsx

```diff
@@ -81,6 +86,16 @@ export const Output: React.FC<OutputProps> = (props: OutputProps) => {
             <Pre className="logOutput">Check out the Stepper tab!</Pre>
           </Card>
         );
+      } else if (props.isHtml) {
+        return (
+          <Card>
+            <Pre className="logOutput">
```

Comment:
```suggestion
            <Pre className="log-output">
```

CSS classnames should use kebab case by convention.